package.json

```diff
@@ -42,11 +42,13 @@
     "gulp-uglify": "1.1.0",
     "gulp-util": "3.0.7",
     "jest-cli": "0.4.18",
+    "jquery": "1.8.3",
     "jsdoc": "^3.3.3",
     "jsdoc-jsx": "^0.1.0",
     "react-addons-test-utils": "0.14.7",
     "react-tools": "*",
     "reactify": "1.1.1",
+    "underscore": "1.3.3",
```

Comment:
added to project to test controller ONLY.

underscore is not bundled into application.